src/Features/Core/Portable/Navigation/NavigableItemFactory.DeclaredSymbolNavigableItem.cs

```diff
@@ -45,6 +45,7 @@ private static Glyph GetPublicGlyph(DeclaredSymbolInfoKind kind)
                 switch (kind)
                 {
                     case DeclaredSymbolInfoKind.Class: return Glyph.ClassPublic;
+                    //case DeclaredSymbolInfoKind.Record: return Glyph.RecordPublic; // I'm not sure what a Glyph is? this I add "RecordPublic" to the enum?
```

Comment:
```suggestion
                    case DeclaredSymbolInfoKind.Record: return Glyph.ClassPublic;
```

Reply:
yup.  Add RecordPublic to that enum

Reply:
@CyrusNajmabadi What behavior would this introduce? I'm unsure of the other places that will need be updated.

Reply:
> In other words, do we really need to treat records differently here?

Yes, i woudl treat them differntly *within* roslyn.  at the boundaries with components out of our control, we may have to consider what to do.

Reply:
@CyrusNajmabadi Can this be addressed separately?

As I said, I'm not sure of the other places that needs to be updated. Also getting the Glyph from ISymbol is dependent on TypeKind enum currently. That should wait until https://github.com/dotnet/roslyn/issues/47713 is resolved.

Reply:
yes.  can you open an issue to track this.